config/initializers/active_record_encryption.rb

```diff
@@ -32,4 +32,8 @@
   config.active_record.encryption.deterministic_key = ENV.fetch('ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY')
   config.active_record.encryption.key_derivation_salt = ENV.fetch('ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT')
   config.active_record.encryption.primary_key = ENV.fetch('ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY')
+
```

Comment:
This is missing the following config:
```suggestion
config.active_record.encryption.support_sha1_for_non_deterministic_encryption = true
```

Indeed, it seems ActiveRecord encryption on 7.0 was using SHA1 even when the default key generator hash digest algorithm was set to SHA256, so anyone that's been using the development version of Mastodon will have records using SHA1, and they need to be supported.